eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -151,7 +172,7 @@ jobs:
           reports: $(CodeCoverageFilePattern)
           targetdir: $(Build.ArtifactStagingDirectory)\coverage
           reporttypes: Cobertura
-          filefilters: +$(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}\**
+          filefilters: +$(Build.SourcesDirectory)\sdk\$(ServiceDirectory)\**
```

Comment:
... almost everywhere
```suggestion
          filefilters: +$(Build.SourcesDirectory)/sdk/$(ServiceDirectory)/**
```

Reply:
And the line directly above `\coverage` as well 👍 

Just updating it everywhere.